app/models/board.py

```diff
@@ -5,14 +5,20 @@ class Board(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     title: Mapped[str] = mapped_column(nullable=False)
     owner: Mapped[str] = mapped_column(nullable=False)
-    cards = relationship('Card', back_populates='board', cascade='all, delete-orphan')
+    cards: Mapped[list["Card"]] = relationship(back_populates="board")
+
+    # cards = relationship('Card', back_populates='board', cascade='all, delete-orphan')
 
     def to_dict(self):
-        result = {"id": self.id, "title": self.title, "cards": self.cards, "owner": self.owner}
-        cards = []
-        for card in self.cards:
-            cards.append(card.to_dict())
-            result["cards"] = cards
+        # cards = []
+        # for card in self.cards:
+        #     cards.append(card.to_dict())
+        result = {
+            "id": self.id, 
+            "title": self.title, 
+            "cards": len(self.cards),
```

Comment:
why? this going to make multiple calls

Reply:
Oh, I changed this back! I just made it shorter when I was testing the API. I think I forgot to push that change.